packages/hackmcx-client/src/App.js

```diff
@@ -1,26 +1,20 @@
 import logo from './logo.svg';
 import './App.css';
-import {useEffect, useState} from 'react';
-import axios from "axios";
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { BrowserRouter as Router, Route, Routes, Link } from 'react-router-dom';
+import CreatePost from './CreatePost';
 
-function App(){
-  const [example, setExample] = useState('-_-');
-  useEffect(async () => {
-    await axios.get('http://localhost:3003/example').then((response) => {
-      setExample(response.data[0].CURRENT_TIMESTAMP)
-    }).catch((any) => console.log(any))
-  }, [example])
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <p>{example}</p>
-      </header>
-    </div>
-  );
+function App() {
+	return (
+		<div className='App'>
+			<Router>
+				<Routes>
+					<Route exact path='/' exact element={<CreatePost />} />
```

Comment:
Please change to: `<Route exact path='/CreatePost' exact element={<CreatePost />} />`